api/src/pcapi/routes/public/individual_offers/v1/serialization.py

```diff
@@ -57,6 +57,8 @@ class StrEnum(str, enum.Enum):
 
 
 class Accessibility(serialization.ConfiguredBaseModel):
+    """Accessibility to disabled people."""
```

Comment:
J'ai l'impression que c'est "Accessibility for disabilities | for disabled people"

Reply:
![Screenshot 2023-03-10 at 12 04 03](https://user-images.githubusercontent.com/95358853/224299979-dad447a8-bf4f-427c-9a87-2dc72b93ec9a.png)
Proposition de reverso sinon